src/app/hooks/useViewTracker/index.tsx

```diff
@@ -32,13 +34,18 @@ const getComponentViewTracker = (eventTrackingData?: EventTrackingData) => {
     statsDestination,
     campaignID,
     detailedPlacement,
-    optimizely,
+    sendOptimizelyEvents,
     optimizelyMetricNameOverride,
   } = extractATITrackingProps({
     eventTrackingData,
     eventType: VIEW_EVENT,
   });
 
+  const { optimizely } = useContext(OptimizelyContext);
+  const optimizelyVariation = useOptimizelyMvtVariation(
+    OPTIMIZELY_CONFIG.ruleKey,
+  );
```

Comment:
This seems to work ok and not send multiple events. Curious if there is a gotcha I've missed where using this Context + Hook directly in the view tracker is a bad idea?

Saves us having to always pass in `optimizely` with `eventTrackingData`, so seems better to handle it all here?